### ISSUE TITLE
internuclear-forces-in-a-diatomic-molecule_list-of-experiments->experiment1->quizzes

### ISSUE THREAD

**Issue by AmoghTiwari** [virtual-labs/molecular-interactions-iiith#160]
Defect Description: All quizzes have serial number 0
Steps to reproduce: Open this link: http://mi-iiith.vlabs.ac.in/exp1/quiz.php
Expected Result: Have serial numbers 1,2,3,4,5
Actual Result: Serial numbers 0,0,0,0,0
![image](https://user-images.githubusercontent.com/40275901/54491564-848c2b00-48e5-11e9-9c72-44fe27c22bf4.png)

**Comment by shanmukh1608** [comment]
Non-issue, I think, because the numbering appears correctly on my end. Please look at attached screenshot.

![Screenshot from 2019-03-19 15-18-47](https://user-images.githubusercontent.com/44746109/54596296-567c2780-4a5a-11e9-8272-8ef388366fce.png)

**Comment by AmoghTiwari** [comment]
It might be a browser related issue. Which browser are you using?
I am using firefox

**Comment by shradhasehgal** [comment]
Fixed 
Commit ID 9adb1008ec496aa152ee01fd535593e7b3054104
